.github/workflows/build.yml

```
@@ -99,4 +99,5 @@ jobs:
           gh release create "${GITHUB_REF_NAME}" \
             --discussion-category Announcements \
             --notes-file release-note.md \
+            --repo ${GITHUB_REPOSITORY} \
```

```suggestion
            --repo "${GITHUB_REPOSITORY}" \
```

変数は `"` で囲むことにしているので、囲っておきますか。

---

そうだった。。。